pkgs/applications/editors/vscode/extensions/overrides.nix

```diff
@@ -0,0 +1,301 @@
+    };
+  };
+  njpwerner.autodocstring.meta.license = licenses.mit;
+  octref.vetur.meta.license = licenses.mit;
+  redhat.java = {
+    buildInputs = [ jdk ];
+    meta = {
+      license = licenses.epl20;
+      broken = lib.versionOlder jdk.version "11";
+    };
+  };
+  rust-lang.rust-analyzer = callPackage ./rust-analyzer { };
+  pkief.material-icon-theme.meta.license = licenses.mit;
+  pkief.material-product-icons.meta.license = licenses.mit;
+  ryu1kn.partial-diff.meta.license = licenses.mit;
+  scala-lang.scala.meta.license = licenses.mit;
+  serayuzgur.crates.meta.license = licenses.mit;
+  silvenon.mdx.meta.license = licenses.mit;
+  skyapps.fish-vscode.meta.license = licenses.mit;
+  slevesque.vscode-multiclip.meta.license = licenses.mit;
+  stephlin.vscode-tmux-keybinding.meta.license = licenses.mit;
+  stkb.rewrap.meta.license = licenses.asl20;
+  tabnine.tabnine-vscode.meta.license = licenses.mit;
+  tamasfe.even-better-toml.meta.license = licenses.mit;
+  timonwong.shellcheck = buildVscodeMarketplaceExtension {
+    mktplcRef = {
+      name = "shellcheck";
+      publisher = "timonwong";
+      version = "0.19.3";
+      sha256 = "0l8fbim19jgcdgxxgidnhdczxvhls920vrffwrac8k1y34lgfl3v";
+    };
+    nativeBuildInputs = [ jq moreutils ];
+    postInstall = ''
+      cd "$out/$installPrefix"
+      jq '.contributes.configuration.properties."shellcheck.executablePath".default = "${shellcheck}/bin/shellcheck"' package.json | sponge package.json
+    '';
+    meta = {
+      license = licenses.mit;
+    };
+  };
+  tobiasalthoff.atom-material-theme.meta.license = licenses.mit;
+  tomoki1207.pdf.meta.license = licenses.mit;
+  vadimcn.vscode-lldb = callPackage ./vscode-lldb { };
+  valentjn.vscode-ltex = vscode-utils.buildVscodeMarketplaceExtension rec {
+    mktplcRef = {
+      name = "vscode-ltex";
+      publisher = "valentjn";
+      version = "13.1.0";
+    };
+
+    vsix = fetchurl {
+      name = "${mktplcRef.publisher}-${mktplcRef.name}.zip";
+      url = "https://github.com/valentjn/vscode-ltex/releases/download/${mktplcRef.version}/vscode-ltex-${mktplcRef.version}-offline-linux-x64.vsix";
+      sha256 = "1nlrijjwc35n1xgb5lgnr4yvlgfcxd0vdj93ip8lv2xi8x1ni5f6";
+    };
+
+    nativeBuildInputs = [ jq moreutils ];
+
+    buildInputs = [ jdk ];
+
+    postInstall = ''
+      cd "$out/$installPrefix"
+      jq '.contributes.configuration.properties."ltex.java.path".default = "${jdk}"' package.json | sponge package.json
+    '';
+
+    meta = with lib; {
+      license = licenses.mpl20;
+      maintainers = [ maintainers._0xbe7a ];
+    };
+  };
+  viktorqvarfordt.vscode-pitch-black-theme.meta.license = licenses.mit;
+  ms-vsliveshare.vsliveshare = callPackage ./ms-vsliveshare-vsliveshare { };
+  vspacecode.vspacecode.meta.license = licenses.mit;
+  vspacecode.whichkey.meta.license = lib.licenses.mit;
+  xyz.local-history.meta.license = lib.licenses.mit;
+  llvm-org.lldb-vscode = llvmPackages_8.lldb;
```

Comment:
In general I have no clue how this package makes sense as a vscode extension, but it doesn't break anything so I kept it